test/conformance/certificate/utils.go

```diff
@@ -73,7 +73,7 @@ func CreateCertificate(t *testing.T, clients *test.Clients, dnsNames []string) (
 // IsCertificateReady will check the status conditions of the certificate and return true if the certificate is
 // ready.
 func IsCertificateReady(c *v1alpha1.Certificate) (bool, error) {
-	return c.Generation == c.Status.ObservedGeneration && c.Status.IsReady(), nil
+	return c.Generation == c.Status.ObservedGeneration && c.IsReady(), nil
```

Comment:
```suggestion
	return c.IsReady(), nil
```

Reply:
The names of these are a bit misleading. The tests are actually looking for the resource to have been observed and failed (False). !IsReady() will do the wrong thing when the state is Unknown.

It looks like they are corrected in v1, but I'll making them uniform.

Reply:
Oops this comment is in the wrong place. The cert one is fixed